app/models/feedback_reporters/feedback_reporter.rb

```diff
@@ -25,17 +28,36 @@ def description
   end
 
   def send_report!
-    HTTParty.post("#{ArchiveConfig.NEW_BUGS_SITE}#{project_path}",
-                  body: "&xml=#{URI.encode_www_form_component(xml.to_str)}")
+    zoho_resource_client.create_ticket(
+      ticket_attributes: report_attributes,
+    )
+  end
+
+  def report_attributes
```

Comment:
It looks like both kinds of reports have `departmentId` in the `report_attributes`, too. Would it be possible to borrow the approach Elz used for setting the `project_path` and use it for `departmentId`, e.g. add this method here:

```
def department_id
  self.class::DEPARTMENT_ID
end
```

And then set `DEPARTMENT_ID = ArchiveConfig.ABUSE_ZOHO_DEPARTMENT_ID` and `DEPARTMENT_ID = ArchiveConfig.SUPPORT_ZOHO_DEPARTMENT_ID` in the appropriate classes?

Reply:
Currently just stubbing `ArchiveConfig` makes the tests work. If we do this, we'll also need to stub `<class>::DEPARTMENT_ID`s as well.

Reply:
Let's file this under "not worth creating extra work at the moment," then.